docs/diagrams/TagFindSequenceDiagram.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+!include style.puml
+
+box Logic LOGIC_COLOR_T1
+participant ":Logic" as Logic LOGIC_COLOR
+participant "command:FindTagCommand" as Command LOGIC_COLOR
+end box
+
+box Ui UI_COLOR_T1
+participant ":UiManager" as UiManager UI_COLOR
+participant ":MainWindow" as MainWindow UI_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+[-> UiManager : start()
+activate UiManager
+UiManager -> MainWindow : fillInnerParts()
+activate MainWindow
+MainWindow -> Model : getFilteredTagList()
+activate Model
+UiManager <-- Model
```

Comment:
You skipped a layer of return here.

Reply:
Thansk for the catch! Fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/337/commits/a5693908010760480c62d0aafc259fbec123a103